src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -2102,11 +2103,24 @@ public static boolean getCoerceReadConsistencyAll()
         return conf.coerce_read_consistency_all;
     }
 
+    public static Set<String> getCoerceReadConsistencyAllKeyspaces()
+    {
+        return conf.coerce_read_consistency_all_keyspaces;
+    }
+
     public static void setCoerceReadConsistencyAll(boolean value)
     {
         conf.coerce_read_consistency_all = value;
     }
 
+    public static boolean getCoerceReadConsistencyAllForKeyspace(String keyspace) {
+        return conf.coerce_read_consistency_all && conf.coerce_read_consistency_all_keyspaces.contains(keyspace);
```

Comment:
Might be more usable to make `coerce_read_consistency_all_keyspaces` default to "every keyspaces" if empty

Reply:
Side note, if we really wanted to merge this, we would probably unify those two fields into one

Reply:
Yep, makes sense